README.md

```diff
@@ -1,2 +1,34 @@
-# eq-json-version-translator
-Translate eQ JSON to different versions
+# eq-json-version-converter
+This repository has been written to convert v2 Survey Runner JSON (https://github.com/ONSdigital/eq-survey-runner) to v3.
+
+N.B This was written for the e-commerce survey, so although it might convert some other v2 surveys they might not pass validation.
+
+## Setup
+
+This application uses pipenv, please follow the following steps.
+
+```
+pyenv install
+pip install --upgrade pip setuptools pipenv
+pipenv install --dev
+pipenv shell
```

Comment:
We shouldn't need to ask the user to enter into a new shell to complete the conversion